.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: bash
+sudo: true
```

Comment:
Not sure but I think that the current white list aliases are [another](https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json) file. Did you checked there what you are looking for?

Also not sure why you need to write a Python script to call `pip`.  In the worse case I guess that `pip` can be called inside the `Makefile`